space-member.ts

```diff
@@ -0,0 +1,15 @@
+import { Kind_V2, Signer, SpaceMember } from "./nostr.ts";
+import { PublicKey } from "./key.ts";
+import { nowTime } from "./_helper.ts";
+
+export function prepareAddMember(author: Signer, member: string): Error | Promise<SpaceMember> {
+    if (PublicKey.FromString(member) instanceof Error) {
+        return new Error("The member's public key is incorrect.");
```

Comment:
just return the error of `FromString` itself.

Reply:
ok